src/devices/tuya.ts

```diff
@@ -3788,18 +3788,13 @@ const definitions: Definition[] = [
         onEvent: tuya.onEventSetTime,
     },
     {
-        fingerprint: [
-            {
-                modelID: 'TS0601',
-                manufacturerName: '_TZE200_viy9ihs7',
-            },
-        ],
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_viy9ihs7']),
         model: 'ZWT198',
         vendor: 'TuYa',
-        description: 'ONNDO Smart thermostat',
+        description: 'ONNDO smart thermostat',
         fromZigbee: [tuya.fz.datapoints],
         toZigbee: [tuya.tz.datapoints],
-        onEvent: tuya.onEventSetLocalTime,
+        onEvent: tuya.onEvent({timeStart: '2000'}),
```

Comment:
May i ask you why you changed this? With the other variant (onEventSetTime if i recall) i had the problem that the thermostat was resetting the date/time that's why i used the onEventSetLocalTime